### PR TITLE
add is_master_worker api

### PR DESCRIPTION
**Pull request by you-n-g**
* add is_master_worker api
* User don't have to worry about initializing values when using MVSharedVariable
* related document is added.

### CODE REVIEW

File: binding/python/README.md [hunk 1]
```diff
@@ -143,7 +167,7 @@ if "THEANO_FLAGS" not in os.environ:
 Here is the result of running [Deep_Residual_Learning_CIFAR-10](./examples/theano/lasagne/Deep_Residual_Learning_CIFAR-10.py)
 
 ## Task Description
-Perform CIFAR-10 classification with residential networking implementation based on Lasagne.
+Perform CIFAR-10 classification with residual networking implementation based on Lasagne.
```
Comment (huxuan) [inline binding/python/README.md]: `residual networks`?

**Comment by huxuan** [comment]
Actually, I am a little concern about the implementation of `mv.is_master_worker()` api, cause whenever we call the `mv.is_master_worker()`, it will lead to a call to the shared object, which is not a good approach from my point of view. Maybe we can make it as a member variable os `multiverso` module?

**Comment by you-n-g** [comment]
Are you concerning about the performance? I don't think this will bring noticeable overhead.
And the benefit is that users don't have to set a value to a module now.

File: binding/python/README.md (hunk 1 above)
Comment (you-n-g) [inline binding/python/README.md]: Amended now

**Pull request merged**